ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/searchUtils.tsx

```diff
@@ -78,7 +78,8 @@ export function parseQuerySearchFilter(rawSearchFilter: SearchFilter): QuerySear
             }
         });
 
-        cleanSearchFilter.Fixable = cleanFixable;
+        // TODO We are explicitly excluding "Fixable" from the search filter until this functionality is re-enabled
```

Comment:
This, along with the `<FilterChips>` change, will prevent a fixable value in the URL from taking effect on the pages. Probably overkill though...